CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ This is the log of notable changes to EAS CLI and related packages.
 
 ### 🎉 New features
 
+- set runtime version in eas update:configure. ([#811](https://github.com/expo/eas-cli/pull/811) by [@jkhales](https://github.com/jkhales))
```

Comment:
```suggestion
- Set runtime version in eas update:configure. ([#811](https://github.com/expo/eas-cli/pull/811) by [@jkhales](https://github.com/jkhales))
```

Reply:
FYI: I just made the changelog bot automatically capitalize the message - https://github.com/expo/eas-cli/commit/9280dfb3adb6e118fe41621caf862253362075a9